lib/doorkeeper/openid_connect/helpers/controller.rb

```diff
@@ -7,7 +7,8 @@ module Controller
         def authenticate_resource_owner!
           super.tap do |owner|
             next unless valid_controller_path?
-            next unless scopes.include?('openid')
```

Comment:
We no longer have `pre_auth.scopes.include?('openid')` assuming that
- pre_auth.scopes exists, and implicitly through `build_scopes` in `PreAuthorization`, that
- pre_auth.client exists

Reply:
Remember to leave this comment on the PR for the OIDC base gem:

"Let us know if Doorkeeper's `PreAuthorization` is the preferred way to check scopes, and why it is so.  `params` should always be available in this context and allows us to avoid the validation issues."